src/calculator/Makefile

```diff
@@ -0,0 +1,8 @@
+EXE=calc
+APP=calculator
+FILES=main.cpp calc.cpp
+DRAFT=calc.draft
+
+include ../actions.make
+
```

Comment:
```suggestion
export CPP_FLAGS += -static
```

Reply:
when i ran this on rM1, i got errors about glibc 2.29 not being on there. making static build fixes that issue for me

Reply:
I've got a bit of an issue with this one - if I add it, it fails to display anything except a blank screen(?)

Reply:
ok, let's not worry about it for now then. i can check what happens when i do static build and run on rM2. what compiler are you using?

re: the symbols: i did an objdump and looked for any symbols in GLIBC 2.29 or greater and found stuff like pow, log, etc, but i know they are in earlier versions of glibc. this is somehow using a newer versions of the API - maybe there is way to force compat version when compiling against glibc?

Reply:
I think those functions have been around forever - I'm using "make rpncalc_docker" to build it, so whichever version that's built with. I just checked and it says:

root@c6c7bac3e76a:/# arm-linux-gnueabihf-g++ --version
arm-linux-gnueabihf-g++ (crosstool-NG 1.24.0) 8.3.0

Reply:
the docker build works for me, thanks, but i still wanted to fix it for my own local dev. 

i did an objdump and found things depending on 2.29, then moved them to 2.4. 

<details>

```
okay@chalk:~/tonka/src/remarkable/rmkit$ objdump -T src/build/rpncalc | grep GLIBC_
00000000      DF *UND*  00000000  GLIBC_2.4   signal
00000000  w   DF *UND*  00000000  GLIBC_2.4   pthread_mutex_lock
00000000      DF *UND*  00000000  GLIBC_2.4   sinh
00000000      DF *UND*  00000000  GLIBC_2.4   sqrt
00000000      DF *UND*  00000000  GLIBC_2.4   __memcpy_chk
00000000      DF *UND*  00000000  GLIBC_2.4   socketpair
00000000      DF *UND*  00000000  GLIBC_2.4   ldexpf
00000000      DF *UND*  00000000  GLIBC_2.4   strtol
00000000      DF *UND*  00000000  GLIBC_2.4   free
00000000      DF *UND*  00000000  GLIBC_2.4   fseek
00000000      DO *UND*  00000000  GLIBC_2.4   __stack_chk_guard
00000000      DF *UND*  00000000  GLIBC_2.4   strncmp
00000000      DF *UND*  00000000  GLIBC_2.4   fgetc
00000000      DF *UND*  00000000  GLIBC_2.4   ceilf
00000000      DF *UND*  00000000  GLIBC_2.4   exit
00000000      DF *UND*  00000000  GLIBC_2.4   cosf
00000000      DF *UND*  00000000  GLIBC_2.4   feof
00000000      DF *UND*  00000000  GLIBC_2.4   cos
00000000      DF *UND*  00000000  GLIBC_2.4   __fprintf_chk
00000000      DF *UND*  00000000  GLIBC_2.7   __fread_chk
00000000      DF *UND*  00000000  GLIBC_2.4   memset
00000000      DF *UND*  00000000  GLIBC_2.4   acosf
00000000      DF *UND*  00000000  GLIBC_2.4   close
00000000      DF *UND*  00000000  GLIBC_2.4   fopen
00000000      DF *UND*  00000000  GLIBC_2.4   read
00000000      DF *UND*  00000000  GLIBC_2.29  log
00000000      DF *UND*  00000000  GLIBC_2.4   abort
00000000      DF *UND*  00000000  GLIBC_2.4   memmove
00000000      DF *UND*  00000000  GLIBC_2.4   log10
00000000      DF *UND*  00000000  GLIBC_2.4   ioctl
00000000      DF *UND*  00000000  GLIBC_2.4   realloc
00000000      DF *UND*  00000000  GLIBC_2.4   mmap
00000000      DF *UND*  00000000  GLIBC_2.4   __assert_fail
00000000      DF *UND*  00000000  GLIBC_2.4   cosh
00000000      DF *UND*  00000000  GLIBC_2.4   strtod
00000000  w   DF *UND*  00000000  GLIBC_2.4   __pthread_key_create
00000000      DF *UND*  00000000  GLIBC_2.4   open
00000000      DO *UND*  00000000  GLIBC_2.4   stderr
00000000      DF *UND*  00000000  GLIBC_2.4   fmod
00000000      DF *UND*  00000000  GLIBC_2.4   fread
00000000      DF *UND*  00000000  GLIBC_2.4   ungetc
00000000      DF *UND*  00000000  GLIBC_2.4   __stack_chk_fail
00000000      DF *UND*  00000000  GLIBC_2.4   sqrtf
00000000      DF *UND*  00000000  GLIBC_2.4   select
00000000      DF *UND*  00000000  GLIBC_2.4   calloc
00000000      DF *UND*  00000000  GLIBC_2.4   ftell
00000000      DF *UND*  00000000  GLIBC_2.4   ferror
00000000      DF *UND*  00000000  GLIBC_2.4   isspace
00000000      DF *UND*  00000000  GLIBC_2.4   fwrite
00000000      DF *UND*  00000000  GLIBC_2.4   memcpy
00000000      DF *UND*  00000000  GLIBC_2.4   tan
00000000      DF *UND*  00000000  GLIBC_2.4   malloc
00000000      DF *UND*  00000000  GLIBC_2.4   strlen
00000000      DF *UND*  00000000  GLIBC_2.4   __memset_chk
00000000      DF *UND*  00000000  GLIBC_2.27  powf
00000000      DF *UND*  00000000  GLIBC_2.4   tanh
00000000      DF *UND*  00000000  GLIBC_2.4   fclose
00000000      DF *UND*  00000000  GLIBC_2.4   write
00000000      DF *UND*  00000000  GLIBC_2.15  __fdelt_chk
00000000      DF *UND*  00000000  GLIBC_2.4   floorf
00000000      DF *UND*  00000000  GLIBC_2.4   fmodf
00000000      DF *UND*  00000000  GLIBC_2.4   pthread_create
00000000      DF *UND*  00000000  GLIBC_2.4   strcmp
00000000      DF *UND*  00000000  GLIBC_2.4   __errno_location
00000000      DF *UND*  00000000  GLIBC_2.29  pow
00000000      DF *UND*  00000000  GLIBC_2.4   memcmp
00000000      DF *UND*  00000000  GLIBC_2.4   __sprintf_chk
00000000      DF *UND*  00000000  GLIBC_2.4   sin
00000000      DF *UND*  00000000  GLIBC_2.4   getenv
00000000      DF *UND*  00000000  GLIBC_2.4   __libc_start_main
00000000  w   DF *UND*  00000000  GLIBC_2.4   __cxa_finalize
00000000      DF *UND*  00000000  GLIBC_2.4   frexpf
00000000      DF *UND*  00000000  GLIBC_2.4   __tls_get_addr
00000000  w   DF *UND*  00000000  GLIBC_2.4   pthread_mutex_unlock
00000000      DF *UND*  00000000  GLIBC_2.4   vsnprintf
00000000      DF *UND*  00000000  GLIBC_2.4   floor


```
</details>

```
// adding this fixes things for me
__asm__(".symver pow,pow@GLIBC_2.4");
__asm__(".symver log,log@GLIBC_2.4");
```